src/main/java/io/molnarsandor/trelloclone/project_task/model/BacklogEntity.java

```diff
@@ -0,0 +1,33 @@
+package io.molnarsandor.trelloclone.project_task.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.molnarsandor.trelloclone.project.model.ProjectEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "backlog")
+@Getter
+@Setter
+@NoArgsConstructor
+public class BacklogEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Integer ptSequence = 0;
+    private String projectIdentifier;
+
+    @OneToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "project_id", nullable = false)
+    @JsonIgnore
+    private ProjectEntity project;
+
+    @OneToMany(cascade = CascadeType.REFRESH, fetch = FetchType.EAGER, mappedBy = "backlog", orphanRemoval = true)
```

Comment:
**EAGER**-t csak akkor használj, ha minden egyes entitás lekérdezésnél fogod is ezt a mezőt használni, ha nem akkor rakd át nyugodtan **LAZY**-re.

A baj az, hogy minél nagyobb a lekérdezendő kollekció mérete, annál nagyobb lesz a resultSet és mivel itt cross join történik, eléggé le tud lassulni a query.